build.sbt

```diff
@@ -189,23 +211,6 @@ lazy val dumpPersistenceSchemaCompilerPlugin = (projectMatrix in file("dump-pers
         }
```

Comment:
Actually... would it make sense to mark `scalaPluginDeps21x` as `Provided`? 🤔

Reply:
So as to avoid packing at least `scala.*` classes into the fat jar

Reply:
you mean these ones?
![image](https://user-images.githubusercontent.com/49597713/174625843-d6c4e59b-3f4f-4bfc-880e-4d2c64b1777f.png)
I can try to change it if you wish